src/ProductConstructionService/ProductConstructionService.Api/Controllers/PcsJobsProcessorController.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Mvc;
+using ProductConstructionService.Api.Queue;
+
+namespace ProductConstructionService.Api.Controllers;
+
+[Route("pcsJobsProcessor")]
```

Comment:
Adding status sounds like a nice idea, but for our deployments, maybe it's ok to wait for a 200 response from `stop`, meaning the processing has been stopped, rather than busy waiting, checking the status

Reply:
Oh, I didn't notice how it's implemented. That's wrong - we shouldn't rely on having a 15 minute long living HTTP requests. These will get cancelled.

We should just set the stop working and return 200. The client should then poll for state which will be Stopping / Processing job and then eventually stopped